### PR TITLE
Unused variable cleanup and new overmask px default

### PR DESCRIPTION
**Pull request by seijihariki**
Default overmask px now 20px as discussed in #125. This pull will also include inpainting model detection and offer to change on first boot, also from #125.

Will probably add tool settings persistence as well at some point. Not necessarily in this pull, but maybe the next.

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -132,35 +132,9 @@ var stableDiffusionData = {
...
-var drawTargets = []; // is this needed?  i only draw the last one anyway...
-var dropTargets = []; // uhhh yeah similar to the above but for arbitrary dropped images
-var arbitraryImage;
-var arbitraryImageData;
-var arbitraryImageBitmap;
-var arbitraryImageBase64; // seriously js cmon work with me here
-var placingArbitraryImage = false; // for when the user has loaded an existing image from their computer
-var marchOffset = 0;
-var inProgress = false;
-var marchCoords = {};
```
Comment (zero01101) [inline js/index.js:163]: goodbye all my horrible global variables, you shan't be missed

**Review by zero01101 — APPROVED** [review]
sir yes sir

**Comment by seijihariki** [comment]
Fucked up the branch, will fix tomorrow.

**Comment by seijihariki** [comment]
Fixed branch, give it a look and merge if applicable! (LMB now also works for pan)

@zero01101 if you could test if it is at least useable using a MacBook touchpad!

**Comment by seijihariki** [comment]
Supposedly should work better for touchpad users in general now, and should mostly resolve #116

**Comment by zero01101** [comment]
confirmed _significantly_ better on macbook touchpad :) ctrl (NOT CMD) + two-finger-vertical swipe zooms beautifully, and ctrl+left-click-drag pans very nicely

**Comment by seijihariki** [comment]
Will merge this pull then for now!

**Pull request merged**